envergo/hedges/views.py

```diff
@@ -35,6 +37,27 @@ def get_object(self, queryset=None):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
+        context["hedge_to_plant_data_form"] = HedgeToPlantPropertiesForm(
```

Comment:
Ça fait beaucoup de lignes pour spécifier deux variables. Rien à dire sur le mécanisme, ça me semble nickel, mais est-ce que ça ne pourrait pas être avantageusement déporté dans des méthodes dédiées ?

```suggestion
        context["hedge_to_plant_data_form"] = self.get_hedge_to_plant_data_form()
        context["hedge_to_remove_data_form"] = self.get_hedge_to_remove_data_form()
```

En l'état je trouve que la méthode `get_context_data` est déjà pas mal difficile à lire.